### ISSUE TITLE
How adjust timezone?

### ISSUE THREAD

**Issue by CountCypher** [LycheeOrg/Lychee#706]
### Detailed description of the problem
Hi all! Maybe it's a dumb question, but how do I adjust the timezone? After the migration from v3 to v4, the timezone is set to UTC. I'd like to adjust the timezone to UTC plus 2 hours (Germany). I've tried to do this in `config/app.php` as suggested in https://lycheeorg.github.io/docs/installation.html#configuration, but didn't succeed.

### Steps to reproduce the issue
1. Login to Lychee
2. Check the log: Timestamps show UTC time
3. Logout from Lychee
4. Edited `config/app.php` for UTC plus 2 hours: `'timezone' => env('TIMEZONE','UTC+2'),`
5. Login to Lychee
6. Check the log: Timestamps still show UTC time

### Output of the diagnostics
```
Diagnostics
-----------
Info: Latest version of PHP is 7.4
Warning: Using 32 bit PHP, recommended upgrade to 64 bit
Warning: '/var/www/Lychee/public/dist/user.css' does not exist or has insufficient read/write privileges.
Warning: '/var/www/Lychee/public/dist/' has insufficient read/write privileges.
Warning: Dropbox import not working. dropbox_key is empty.
Warning: You may experience problems when uploading a large amount of photos. Take a look in the FAQ for details.

System Information
------------------
Lychee Version (release):   4.0.6
DB Version:                 4.0.6

composer install:           --no-dev
APP_ENV:                    production
APP_DEBUG:                  false

System:                     Linux
PHP Version:                7.3
MySQL Version:              10.3.23-MariaDB-0+deb10u1

Imagick:                    1
Imagick Active:             1
Imagick Version:            1690
GD Version:                 2.2.5

Config Information
------------------
version:                    040006
check_for_updates:          0
sorting_Photos_col:         takestamp
sorting_Photos_order:       ASC
sorting_Albums_col:         id
sorting_Albums_order:       DESC
imagick:                    1
skip_duplicates:            0
small_max_width:            0
small_max_height:           360
medium_max_width:           1920
medium_max_height:          1080
lang:                       de
layout:                     0
image_overlay:              0
image_overlay_type:         exif
default_license:            none
compression_quality:        90
full_photo:                 1
delete_imported:            1
Mod_Frame:                  1
Mod_Frame_refresh:          30
thumb_2x:                   1
small_2x:                   1
medium_2x:                  1
landing_page_enable:        0
landing_owner:              John Smith
landing_title:              John Smith
landing_subtitle:           Cats, Dogs & Humans Photography
landing_facebook:           https://www.facebook.com/JohnSmith
landing_flickr:             https://www.flickr.com/JohnSmith
landing_twitter:            https://www.twitter.com/JohnSmith
landing_instagram:          https://instagram.com/JohnSmith
landing_youtube:            https://www.youtube.com/JohnSmith
landing_background:         dist/cat.jpg
site_title:                 Lychee v4
site_copyright_enable:      1
site_copyright_begin:       2019
site_copyright_end:         2019
additional_footer_text:     
display_social_in_gallery:  0
public_search:              0
SL_enable:                  0
SL_for_admin:               0
public_recent:              0
recent_age:                 1
public_starred:             0
downloadable:               0
photos_wraparound:          1
map_display:                0
zip64:                      1
map_display_public:         0
map_provider:               Wikimedia
force_32bit_ids:            0
map_include_subalbums:      0
update_check_every_days:    3
has_exiftool:               0
share_button_visible:       0
import_via_symlink:         0
has_ffmpeg:                 1
location_decoding:          0
location_decoding_timeout:  30
location_show:              1
location_show_public:       0
rss_enable:                 0
rss_recent_days:            7
rss_max_items:              100
prefer_available_xmp_metadata: 0
```

### Browser and system
Client: Firefox 78.2.0esr on Fedora 32 Workstation
Server: Debian Buster on armv7l

**Comment by d7415** [comment]
To start, I'd suggest you try `Europe/Berlin` as `UTC+2` is not a [supported timezone](https://www.php.net/manual/en/timezones.php) in PHP. It may also be worth setting that in `.env` instead of `config/app.php` (I'm about to update the docs for that).

**Comment by CountCypher** [comment]
Thanks for the hint. So I've edited the `config/app.php` accordingly...
`'timezone' => env('TIMEZONE','Europe/Berlin'),`
...but the time of the log timestamps wasn't affected by it.

Then I set the timezone entry in `config/app.php` back to `'timezone' => env('TIMEZONE','UTC'),` and made this entry in `.env`...
`TIMEZONE=Europe/Berlin`
...and that seemed to work then (time in log timestamps is now correct).

I find it a bit confusing that this fundamental setting can be set in two places. As a non-developer, it's also not clear to what the linkage or dependencies between `.env` and `config/app.php` exactly are. Maybe a documentation update would indeed help here. ;) Thanks for the quick help! :)
